documentation/Getting-started-with-TypeSpec-specifications.md

```diff
@@ -16,10 +16,10 @@ Within the azure-rest-api-specs repository, the TypeSpec specification serves as
 
 #### ARM:
 - :white_check_mark: TypeSpec folder name should end with `.Management`.
-- :white_check_mark: This `tspconfig.yaml` is [standard for ARM](https://github.com/Azure/typespec-azure/blob/main/eng/feeds/arm/tspconfig.yaml). The one thing you may customize is the output json file name for `typespec-autorest` emitter.
+- :white_check_mark: Starting from `tspconfig.yaml`[ARM template](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml). The one thing you may customize is the output json file name for `typespec-autorest` emitter.
 
```

Comment:
Are these part of your change? Can you clean this up and only have the changes you want to make show up here?